Assignment 3.Rmd

```diff
@@ -0,0 +1,218 @@
+---
+title: "Assignment 3: K Means Clustering"
+---
+
+In this assignment we will be applying the K-means clustering algorithm we looked at in class. At the following link you can find a description of K-means:
+
+https://www.cs.uic.edu/~wilkinson/Applets/cluster.html
+
+
+```{r}
+library(tidyr)
+
+
+```
+
+Now, upload the file "Class_Motivation.csv" from the Assignment 3 Repository as a data frame called "K1""
+```{r}
+
+K1 <- read.csv("HUDK405018-cluster.csv", header=TRUE, na.strings="NA")
+P1 <- read.csv("Class_Motivation.csv", header=TRUE, na.strings="NA")
+
+```
+
+This file contains the self-reported motivation scores for a class over five weeks. We are going to look for patterns in motivation over this time and sort people into clusters based on those patterns.
+
+But before we do that, we will need to manipulate the data frame into a structure that can be analyzed by our clustering algorithm.
+
+The algorithm will treat each row as a value belonging to a person, so we need to remove the id variable.
+
+```{r}
+library(dplyr)
+K2 <- slice(K1, -1:-2)#return df with all except 1 2 rows
```

Comment:
Nice use of a comment. It's good practice if you want to look back at this as a reference later on.